False positive on opening_hours with a ruleset override

Parser detects an incorrect opening_hours on `Tu-Fr 11:00-19:00; Sa 10:00-22:15; 14:00-15:00 closed`

It seems valid on https://wiki.openstreetmap.org/wiki/Key:opening_hours#Syntax since it's just a ruleset overriding a previous ruleset.

[Openingh](https://openingh.openstreetmap.de/evaluation_tool/?EXP=Tu-Fr%2011%3A00-19%3A00%3B%20Sa%2010%3A00-22%3A15%3B%2014%3A00-15%3A00%20closed&lat=48.7769&lon=9.1844&mode=0&DATE=1664152020000) is ok, [YoHours](https://projets.pavie.info/yohours/?oh=Tu-Fr%2011:00-19:00;%20Sa%2010:00-22:15;%2014:00-15:00%20closed) doesn't display anything.

Object :
* https://www.openstreetmap.org/node/4579458237
* http://localhost:8111/import?url=http://osmose.openstreetmap.fr/api/0.3/issue/dc81afc2-8ebe-2403-8024-ba2b8bdb3ed4/fix/0

(maybe duplicate of #1106 )

Done but I'm still having problems testing the change.
I do
```
cd docker
docker-compose -f docker-compose.yml -f docker-compose-dev.yml build
docker-compose -f docker-compose.yml -f docker-compose-dev.yml run backend bash
```
and then `pytest -k Opening`, which is IIRC what worked to run the tests in `plugins/TagFix_Opening_Hours.py`,
but I get this error from pytest:

```

================================================================================================== ERRORS ===================================================================================================
____________________________________________________________________________________ ERROR collecting modules/OsmPbf.py _____________________________________________________________________________________
ImportError while importing test module '/opt/osmose-backend/modules/OsmPbf.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
modules/OsmPbf.py:22: in <module>
    from . import OsmPbf_libosmbf
modules/OsmPbf_libosmbf.py:24: in <module>
    from .osm_pbf_parser import osm_pbf_parser
E   ImportError: cannot import name 'osm_pbf_parser' from 'modules.osm_pbf_parser' (/opt/osmose-backend/modules/osm_pbf_parser/__init__.py)
```
(and the same from modules/OsmPbf_libosmbf.py)

This seems completely unrelated to my changes, so I must be doing something wrong?

Done but I'm still having problems testing the change.
I do
```
cd docker
docker-compose -f docker-compose.yml -f docker-compose-dev.yml build
docker-compose -f docker-compose.yml -f docker-compose-dev.yml run backend bash
```
and then `pytest -k Opening`, which is IIRC what worked to run the tests in `plugins/TagFix_Opening_Hours.py`,
but I get this error from pytest:

```

================================================================================================== ERRORS ===================================================================================================
____________________________________________________________________________________ ERROR collecting modules/OsmPbf.py _____________________________________________________________________________________
ImportError while importing test module '/opt/osmose-backend/modules/OsmPbf.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
modules/OsmPbf.py:22: in <module>
    from . import OsmPbf_libosmbf
modules/OsmPbf_libosmbf.py:24: in <module>
    from .osm_pbf_parser import osm_pbf_parser
E   ImportError: cannot import name 'osm_pbf_parser' from 'modules.osm_pbf_parser' (/opt/osmose-backend/modules/osm_pbf_parser/__init__.py)
```
(and the same from modules/OsmPbf_libosmbf.py)

This seems completely unrelated to my changes, so I must be doing something wrong?

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,7 +9,7 @@ transporthours
 pyproj >= 2.1.0
 Unidecode
 osmium >= 3.1.3
-git+https://invent.kde.org/libraries/kopeninghours.git@v21.12.3
+git+https://invent.kde.org/libraries/kopeninghours.git@v22.08.3
 git+https://github.com/jocelynj/PyEasyArchive.git
 protobuf < 4 # 4.x binary not yet compatible with system package, deps of vt2geojson
 vt2geojson

